tools/pythonpkg/tests/fast/udf/test_scalar.py

```diff
@@ -127,9 +127,11 @@ def no_op(x):
         con = duckdb.connect()
         map_type = con.map_type('VARCHAR', 'BIGINT')
         con.create_function('test_map', no_op, [map_type], map_type, type=udf_type)
-        rel = con.sql("select test_map(map(['non-inlined string', 'test', 'duckdb'], [42, 1337, 123]))")
-        res = rel.fetchall()
-        assert res == [({'key': ['non-inlined string', 'test', 'duckdb'], 'value': [42, 1337, 123]},)]
+        for arrow_large_buffer_size in [False, True]:
```

Comment:
Im confused, this is not testing the Arrow export?
Can you revert this change and add/change a dedicated test for outputting maps with `.arrow()`

Likely found in tests/fast/arrow

Reply:
Added a new test `tests/fast/arrow/test_10795.py`